src/core/public/chrome/ui/header/header.tsx

```diff
@@ -220,6 +224,13 @@ export function Header({
                   loadingCount$={observables.loadingCount$}
                 />
               </EuiHeaderSectionItem>
+              <EuiHeaderSectionItem border="right">
+                <RecentItems
+                  recentlyAccessed$={observables.recentlyAccessed$}
+                  workspaceList$={observables.workspaceList$}
+                  navigateToUrl={application.navigateToUrl}
+                />
+              </EuiHeaderSectionItem>
             </EuiHeaderSection>
```

Comment:
should this protected by nav group flag?

Reply:
Yes, added.